tests/utils/test_ensembl_rest_clients.py

```diff
@@ -1,60 +1,106 @@
-# -*- coding: UTF-8 -*-
-import tempfile
-from urllib.error import HTTPError
-from urllib.parse import urlencode
-from scout.utils import ensembl_rest_clients as eracs
+"""Tests for ensembl rest api"""
 
+import responses
+from requests.exceptions import HTTPError, MissingSchema
 
-def test_ping_ensemble_37():
+from scout.utils import ensembl_rest_clients
+
+
+@responses.activate
```

Comment:
What does this activate do?

Reply:
Never mind, I've read the documentation!

Reply:
Oh this is one of the reasons I switched to use requests. As you know it is the most used third party library for python so there are a lot of extra stuff available. This responses is a test library for requests. This is how you activate the responses in a test